test/go.mod

```
@@ -134,9 +134,9 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gotest.tools/v3 v3.5.0 // indirect
-	k8s.io/apiserver v0.29.5 // indirect
+	k8s.io/apiserver v0.29.6 // indirect
 	k8s.io/cluster-bootstrap v0.29.3 // indirect
```

```suggestion
	k8s.io/cluster-bootstrap v0.29.6 // indirect
```
this probably needs to be done manually.